common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -8,122 +8,30 @@ Cylindrical Fence
-+------------+--------------------+-------------------+
-+     4      | BRAKE/LAND         |SmartRTL/HOLD      +
-+------------+--------------------+-------------------+
-+     5      | SmartRTL or  LAND  | not applicable    +
-+------------+--------------------+-------------------+
-
-.. note:: RTL/HOLD,etc. means it will attempt RTL first, but if it can't switch to that mode (ie for RTL, GPS lock is lost), it will attempt to switch to the next,in this case HOLD.
-
-At the moment the fence is breached a backup fence is erected 20m
-further out (or up).  If the vehicle breaches this backup fence (for
-example if the vehicle is not set up correctly or the operator takes
-control but is unable to bring the copter back towards home), the vehicle
-will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` again (and another backup fence an additional
-20m further out will be created if this is breached again).
-
-If the vehicle eventually proceeds 100m outside the configured fence
-distance, despite the backup fences, the vehicle will switch into LAND mode (HOLD for Rover).  The idea being that it's clearly impossible to get the vehicle home so best to just bring it
-down/stop it.  The pilot can still retake control of course with the flight mode
-switches.  Like with the earlier fences, another fence is erected 20m
-out which will again switch the vehicle to LAND (HOLD for Rover), if it continues away from
-home.
-
-Plane Actions
--------------
-
-If, enabled, and  the fence radius (:ref:`FENCE_RADIUS<FENCE_RADIUS>`) or altitudes, if non-zero, (:ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` and :ref:`FENCE_ALT_MIN<FENCE_ALT_MIN>`) are exceeded, Plane will execute the :ref:`FENCE_ACTION<FENCE_ACTION>` below:
-
-+------------+--------------------+
-+FENCE_ACTION|  Plane             +
-+============+====================+
-+     0      |   Report Only      +
-+------------+--------------------+
-+     1      | RTL                +
-+------------+--------------------+
-+     6      | Guided to return   +
-+            | point              +
-+------------+--------------------+
-+     7      |  Guided to return  +
-+            |  point with pilot  +
-+            |  throttle control  +
-+------------+--------------------+
-
-If :ref:`FENCE_ACTION<FENCE_ACTION>` is set to a guided return mode (6 or 7), the altitude at which it returns is determined by :ref:`FENCE_RET_ALT<FENCE_RET_ALT>`. If a fence return point is not setup(see video below) for these actions, the vehicle will RTL.
-
-The return point can also be replaced by the nearest rally point (See: :ref:`common-rally-points`) as a destination, if :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled.
+ArduPilot includes a simple "tin can" shaped fence centered on home that will attempt to stop your copter/rover from flying/driving too far away by stopping at the fence in some Copter/Rover modes if :ref:`common-object-avoidance-landing-page` is setup and executing a configurable action.
```

Comment:
Hi, txs for this.  what does "executing a configurable action" mean? I would chop this part.

Reply:
will do